hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rest/RestClient.java

```diff
@@ -112,7 +113,7 @@
   private final URI ozoneRestUri;
   private final CloseableHttpClient httpClient;
   private final UserGroupInformation ugi;
-  private final OzoneAcl.OzoneACLRights userRights;
+  private final ACLType userRights;
```

Comment:
We are going to remove the REST client. So this might be wasted work.

Reply:
change is for compiling purpose only. We can remove REST client.